src/capability/access.js

```diff
@@ -0,0 +1,24 @@
+import { Base } from '../base.js'
+
+/**
+ * Client for interacting with the `access/*` capabilities.
+ */
+export class AccessClient extends Base {
+  /**
+   * Authorize the current device to use capabilities granted to email.
+   *
+   * @param {`${string}@${string}`} email
+   * @param {object} [options]
+   * @param {AbortSignal} [options.signal]
+   */
+  async authorize (email, options) {
+    return this._agent.authorize(email, options)
+  }
+
+  /**
+   * Claim delegations
+   */
+  async claim () {
+    return this._agent.claimDelegations()
```

Comment:
fyi: i want to https://github.com/web3-storage/w3protocol/pull/554#discussion_r1139293664

You would then be able to do
```
import { claimDelegations } from `@web3-storage/access/agent`
await claimDelegations(agent, account, { addProofs: true })
```